filters/auth/authclient.go

```diff
@@ -32,6 +32,7 @@ type authClient struct {
 
 type tokeninfoClient interface {
 	getTokeninfo(token string, ctx filters.FilterContext) (map[string]any, error)
+	Close()
```

Comment:
io.Closer has `Close() error` or we could also not expose it and use `close()` ?

Reply:
This is an unexported interface for internal use.
Tokeninfo client has Close() method and we need one to close tokeninfocache so I put it into this interface.
It makes cleanup simpler and unified for both client and cache.